components/automate-cli/pkg/verifyserver/server/routes.go

```diff
@@ -26,5 +26,7 @@ func (vs *VerifyServer) SetupRoutes() {
 	apiStopGroup := apiV1Group.Group("/stop")
 	apiStopGroup.Post("/mock-server", vs.Handler.StopMockServer)
 
+	apiChecksGroup.Get("/system-user", vs.Handler.CheckSystemUser)
```

Comment:
move it above to apiCheckHGroup api collection

Reply:
Done